templates/kubernetes/7/rancher-compose.yml

```diff
@@ -0,0 +1,83 @@
+.catalog:
+  name: Kubernetes
+  version: v1.3.0-rancher1
+  description: Rancher Kubernetes service
+  minimum_rancher_version: v1.1.0
```

Comment:
Should this version be v1.1.2? or whatever official version pre-release would have